tokens/finn.no/radio.yml

```diff
@@ -5,22 +5,22 @@ color:
   radio:
     background:
       disabled: gray-200
-      hover: blue-50
+      hover: s-color-background-primary-subtle
     border:
-      active:
-        _: blue-600
-        disabled: bluegray-300
-        hover: blue-700
       _: gray-500
-      disabled: bluegray-300
-      hover: blue-600
+      hover: s-color-border-primary
+      selected:
+        _: s-color-border-selected
+        hover: s-color-border-selected-hover
+        disabled: s-color-border-disabled
+      disabled: s-color-border-disabled
     negative:
       background:
-        _: white
-        hover: red-50
+        _: s-color-background
+        hover: s-color-background-negative-subtle
       border:
-        active:
-          _: red-600
-          hover: red-700
-        _: red-600
-        hover: red-700
\ No newline at end of file
+        _: s-color-border-negative
+        hover: s-color-border-negative-hover
+        selected:
+          _: s-color-border-negative-active
```

Comment:
We've discussed this but wanted to keep it in the context of this PR.

This is how invalid checkbox and radio look with the current color tokens:
![Screenshot 2023-07-10 at 17 10 14](https://github.com/warp-ds/css/assets/41303231/45780143-e477-42e0-9539-cd33ba2cd81b)
![Screenshot 2023-07-11 at 08 54 46](https://github.com/warp-ds/css/assets/41303231/1463bf71-6706-46da-a572-27cb422755a6)

I believe the reason why we'd go with `s-color-border-negative-active` token in radio is that this color is applied to `border-color` CSS property of the element. Still, I think we should consider replacing this darker red (`red-800`), either directly with `red-600` or with `s-color-background-negative` map as in checkbox. cc @adidick